### ISSUE TITLE
Website accessibility and content problems

### ISSUE THREAD

**Issue by chriskrycho** [fsharp/FAKE#2040]
(Please take all of this as it is intended: friendly suggestions for improvement, from someone who quite likes the promise of FAKE and thinks it deserves wider adoption!)

### Description

1. **Accessibility:** the website is a bit difficult to read at best, and has some contrast issues in the main description that will make it borderline or actually inaccessible to anyone with vision challenges.

2. **Page content:**
    - The first call to action a new user sees when landing on the page is a "Learn More" button… which takes the user to a page about *contributing* to FAKE 5.
    - The landing page in general is focused on transition to FAKE 5 rather than simply how to use FAKE.

### Repro steps

1. Visit the webpage as a new FAKE user!

### Expected behavior

1. **Accessibility:** Content should be validated for sufficiently high contrast. WCAG recommends a minimum contrast of 4.5:1 for text in general, and 3:1 for large text. Against the lighter blue parts of the backdrop, it falls well below this. (You can test this for yourself with [this tool](http://www.brandwood.com/a11y/).)

2. **Page content:**
    - The FAKE 5 transition is important, but it's much *less* important than basic "how to get started" content. It should probably be moved to its own dedicated page linked *from* the landing page with a "On FAKE 4? Migrate to FAKE 5!" or similar call to action.
    - The primary focus of the page should be landing page should be helping new users get started – not needing to dig into a menu to find the getting started info.
    - It would be really helpful for the landing page to explain the basic value proposition for FAKE. As it is, the language ("A DSL for build tasks and more; the power of F♯ – anywhere – anytime") doesn't actually tell me why I should use this, especially as compared to any other tool. The front page should be dedicated to the value proposition of the tool: what makes it different than (and better for me!) than Make? than VS project files? than PowerShell scripts?

**Comment by matthid** [comment]
Yes feedback like this is very very important, thanks! It is very hard to get feedback like this for some reason. And yes I agree in principle with most of your points:

>  the website is a bit difficult to read at best, and has some contrast issues in the main description that will make it borderline or actually inaccessible to anyone with vision challenges.

Yes my css/html skills are quite limited as you can see ;). There is also a long outstanding bug in the menu...

> The first call to action a new user sees when landing on the page is a "Learn More" button… which takes the user to a page about contributing to FAKE 5.

Yes indeed

> The landing page in general is focused on transition to FAKE 5 rather than simply how to use FAKE.

Yes, transition will be less and less important over time

> It would be really helpful for the landing page to explain the basic value proposition for FAKE. As it is, the language ("A DSL for build tasks and more; the power of F♯ – anywhere – anytime") doesn't actually tell me why I should use this, especially as compared to any other tool. 

I guess we can have a dedicated page for comparison with each of those other tools you list. What is your suggestion for the landing page?

**The good thing is** all the documentation is generated from this repository and can be edited right from the github page:

- Indexpage: https://github.com/fsharp/FAKE/blob/release/next/help/templates/indexpage.cshtml
- Template: https://github.com/fsharp/FAKE/blob/release/next/help/templates/template.cshtml
- Content: https://github.com/fsharp/FAKE/tree/release/next/help/markdown

Please feel free to just change stuff and send suggestions via PR. Any concrete action we could take to fix any of your "issues" is very welcome (if you cannot send PR for some reason). So please feel free to suggest actions we could take in addition to your analysis of the problem.

**Comment by chriskrycho** [comment]
Awesome – I wasn't sure where the templates were, and I will see if I can make some improvements over the next little while.

Where is all the CSS generated from?

**Comment by matthid** [comment]
The css is just static files: https://github.com/fsharp/FAKE/tree/release/next/help/content/assets/css

Honestly, there is probably some rust/bloat in there but hard to cleanup because no-one has an idea of what is used where (fake is like 10 years old...). We might consider using a more radical approach here if needed - we can always test stuff in our staging environment https://staging.fake.build/ if needed

**Comment by chriskrycho** [comment]
Perfect. I'll see if I can chip away little by little; I'm fairly familiar with the refactoring-old-CSS dance. (The WordPress themes I've seen… 😬)

**Comment by matthid** [comment]
Ok I party addressed this in https://github.com/fsharp/FAKE/commit/d8988b33548567959eacc9a06dcc2d0eedd6829d by changing the learn-more link to https://staging.fake.build/fake-what-is-fake.html (and extending the page)

**Comment by matthid** [comment]
I feel like we solved this now. @chriskrycho Please feel free to leave more feedback if not, very appreciated.

**Comment by chriskrycho** [comment]
This all looks great! I’m very sorry I couldn’t end up chipping in – August through October ended up being the single hardest and most exhausting months of my career so far – but this is both way more informative and way more accessible. Thanks so much for acting so thoroughly on my out-of-nowhere feedback; I really admire it!
